ext/session/session.c

```diff
@@ -46,6 +46,7 @@
 #include "ext/standard/basic_functions.h"
 #include "ext/standard/head.h"
 #include "ext/random/php_random.h"
+#include "ext/random/php_random_csprng.h"
```

Comment:
Where is this old header still needed?

Reply:
In `php_session_gc()` for `php_combined_lcg()`. The CSPRNG should likely not be used in there, because it is needlessly slow. Not that `php_combined_lcg()` is particularly fast.

I hope to replace the use of `php_combined_lcg()` there. Probably by an ext/session-specific instance of xoshiro256**, once I clean up the API further to make the distinct engines easier to use programmatically.

Reply:
ACK